tests/std/tests/VSO_0000000_vector_algorithms/test.cpp

```diff
@@ -1140,6 +1156,7 @@ void test_basic_string_dis(mt19937_64& gen, D& dis) {
     for (;;) {
         input_needle.clear();
 
+        test_case_string_rfind_ch(input_haystack, static_cast<T>(dis(gen)));

```

Comment:
This is defined after `test_case_string_rfind_str`, so I would expect it to be called after it, unless there's a strong reason otherwise.